src/Components/CreateImageWizard/CreateImageWizard.scss

```diff
@@ -90,5 +90,5 @@ div.pf-v5-c-alert.pf-m-inline.pf-m-plain.pf-m-warning {
 
 // Fixes min height issues on the main wizard container
 .pf-v5-c-wizard__outer-wrap {
-    min-height: 602px;
+   height: 50px;
```

Comment:
This works but I wish we could explain why... without knowing why this is working it feels like a bit of a hack.

I'm worried that at some point some upstream css (for instance, in `insights-chrome`) is going to change and we're going to end up with a 50 pixel high wizard! Likely? Probably not. But possible? Maybe...

Reply:
maybe the idea to change it to 50px is good if we remove the section and add pageSection like we have in the imageTable page WDYT @lucasgarfield